### DISCUSSION TITLE
MiniProfiler Integration

### DISCUSSION THREAD

**Discussion by dennis-gr** [nhibernate/nhibernate-core#3285] [Show and tell]
I've created a MiniProfiler integration for NHibernate that (should) work with all drivers. It supports profiling SQL and the query/2nd Level cache. Packages are avaiable [here](https://www.nuget.org/packages/NHibernate.MiniProfiler) for NHibernate 5.0-5.4.

**Comment by fredericDelaporte** [comment]
You may announce it on [nhusers group](https://groups.google.com/g/nhusers). You can also write a [blog post on nhibernate.info](https://nhibernate.info/blog/), by contributing it through a PR [here](https://github.com/nhibernate/nhibernate.github.io/tree/master/_posts).
